### DISCUSSION TITLE
Add a priority to the issues or provide a guideline for deciding it.

### DISCUSSION THREAD

**Discussion by gautam1168** [PostgREST/postgrest#1922] [Q&A] [answered]
Would it be a good idea to have some sort of indicator on the issues for people looking to contribute to this repository that helps them decide which issues need fixing most urgently? I guess one can go by the date of creation of the issue, but then can we add a guideline in the contribution docs saying that you should choose the issues opened most recently?

**Comment by steve-chavez** [comment, marked as answer]
Lately we've been using https://github.com/orgs/PostgREST/projects/2 so anyone can know of the features being developed.

We're not prioritizing issues though. Hm, right now the only issue I can think of urgent is this [one](https://github.com/PostgREST/postgrest/issues/1857) which will cause us to break for pg 14. Most issues have workarounds.

> I guess one can go by the date of creation of the issue, but then can we add a guideline in the contribution docs saying that you should choose the issues opened most recently?

Sounds good. We can mention that we recommend solving most recent [beginner](https://github.com/PostgREST/postgrest/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner) issues.
